app/parse.py

```diff
@@ -15,8 +33,96 @@ class Product:
     num_of_reviews: int
 
 
+PRODUCT_FIELDS = [field.name for field in fields(Product)]
+
+
+def init_driver(headless: bool = True) -> webdriver.Chrome:
+    options = Options()
+    options.headless = headless
```

Comment:
Using the `headless` option in Selenium can sometimes cause issues with websites that have anti-bot measures. If you encounter problems, consider running the browser in non-headless mode for debugging purposes.